domains/dangiprashant.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dangiprashant",
+           "email": "prashantdangi983@gmail.com",
+           "discord": "994524133847212062"
+        },
+    
+        "record": {
+            "A": ["blog"]
```

Comment:
Invalid A record, should be an array containing strings which contain valid IPv4 addresses.